src/components/search/searchPageSearchForm/searchPageSearchForm.module.scss

```diff
@@ -8,6 +8,15 @@
   grid-gap: $spacing-m;
 
   @include breakpoints.respond-above(s) {
-    grid-template-columns: 3fr 1fr;
+    grid-template-columns: repeat(4, 1fr);
+
+    & > *:first-child,
+    & > *:nth-child(2) {
+      grid-column: span 2;
+    }
+  }
+
+  .submitButton {
+    grid-column: 4;
```

Comment:
This should be inside the breakpoint definition above. As it is now, it will break the mobile layout


![image](https://user-images.githubusercontent.com/25960123/130204816-4c702957-299f-4d86-b69f-54c983c00658.png)